frontend/src/components/widgets/FileUploader/FileUploader.tsx

```diff
@@ -111,7 +111,7 @@ class FileUploader extends React.PureComponent<Props, State> {
 
     if (!multipleFiles && this.state.files.length) {
       // Only one file is allowed. Delete existing file.
```

Comment:
Is this comment no longer accurate? What is the difference between "delete" and "removeFile" anyways?

Reply:
It's still accurate. `delete` will make the `delete` HTTP request and `removeFile` just removes it from the screen.